charts/localstack/values.yaml

```diff
@@ -46,16 +46,20 @@ debug: false
 ## extraEnvVars:
 ##   - name: DEFAULT_REGION
 ##     value: "us-east-1"
-extraEnvVars: []
+extraEnvVars:
+  - name: SERVICES
```

Comment:
Thanks for this PR @mttcnnff ! Can we please add a descriptive title, and a short summary of the changes in the PR? Thanks

I think we probably want to remove the `SERVICES` env var here - we should leave the default value, otherwise only Elasticsearch Services would get started. What do you think? Thanks

Reply:
@whummer lol sorry, tbh I didn't mean to create this against the main repo's master branch yet, but since I did it, might as well follow through 😁

Reply:
Also i can rebase to pick up the change from PR [#8](https://github.com/localstack/helm-charts/pull/8)

Reply:
@whummer rebased, updated based on the PR #8, and added some description in the README for this PR!!

LMK if you need anything 👍

Reply:
Also in regards to:
> I think we probably want to remove the SERVICES env var here - we should leave the default value, otherwise only Elasticsearch Services would get started. What do you think? Thanks

I actually think we should leave it. Before my PR elastic search was not being started up by default but was being exposed for some reason... To me it makes more sense to just follow through with exposing it and have it running.

That way there's a clear example of how to spin up a service. I'd expect users to almost always supply their own values to customize which api services are running as just running the edge service isn't really useful at all.

Reply:
Thanks for the update @mttcnnff . The thing is - leaving the `SERVICES` config blank means "start up all services" (which is the default). I think if the user is doing a `helm install ...` without any customizations, we should spin up LocalStack in the default configuration as much as possible. Thoughts?

Reply:
Quick follow-up @mttcnnff - can we better remove the `SERVICES` config? Would be great to get this PR merged - thanks!